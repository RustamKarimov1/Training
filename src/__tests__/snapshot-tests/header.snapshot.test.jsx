import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { Header } from '../../components/header/header';

configure({ adapter: new Adapter() });

describe('<Header />', () => {
    it('should render correctly', () => {
        const HEADER = shallow(<Header />);
        expect(shallowToJson(HEADER)).toMatchSnapshot();
    });
});
