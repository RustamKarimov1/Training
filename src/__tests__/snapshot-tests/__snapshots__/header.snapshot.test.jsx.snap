// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<div
  className="header"
>
  <Logo />
  <Connect(SearchField)
    onChange={[Function]}
  />
  <Connect(SearchBar)
    limit={100}
  />
</div>
`;
